engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -582,6 +624,15 @@ public void testExtensionMethod() throws Exception {
       """);
   }
 
+  @Test
+  @Ignore
+  public void testExtensionOperator() throws Exception {
+    parseTest("""
+    Text.* : Integer -> Text
```

Comment:
Right now the combination of `.*` is represented as erroneous in the `Tree`. That will likely need some fixes on the Rust side, @kazcw.